pubspec.yaml

```diff
@@ -5,17 +5,17 @@ homepage: https://github.com/Workiva/over_react_test/
 authors:
   - Workiva UI Platform Team <uip@workiva.com>
 environment:
-  sdk: ">=1.24.2"
+  sdk: ">=1.24.2 <2.0.0"
 dependencies:
   js: ^0.6.1
   matcher: ">=0.11.0 <0.13.0"
-  over_react: ^1.14.0
-  react: ">=3.4.0 <5.0.0"
-  test: ^0.12.24
+  over_react: ^1.23.0
+  react: ">=3.7.0 <5.0.0"
+  test: ^0.12.32+1
```

Comment:
Could raising this dependency contribute to version lock in consumer repos? Is there a reason it's being raised?

Reply:
It is being raised beyond 0.12.32 because 0.12.32 broke smithy.  I'm bumping the lower bound merely to keep things relatively current.